tests/e2e/e2e_test.go

```diff
@@ -73,6 +73,29 @@ func pingSvc(url string) {
 	}
 }
 
+func pingCmd(cmd string) bool {
+	pingTimeout := time.After(5 * time.Minute)
+	tick := time.Tick(time.Second)
+
+	for {
+		select {
+		case <-pingTimeout:
+			Fail("timeout out after 5 minutes")
+
+		case <-tick:
+			out, err := exec.Command("/bin/sh", "-c", cmd).Output()
+			if err != nil {
+				Fail(err.Error())
+			}
+
+			outInt, _ := strconv.Atoi(string(out))
+			if outInt > 0 {
```

Comment:
what happens when `outInt` is 0 or less?

Reply:
@kadel continues in the loop for 5 minutes